integration/src/test/resources/scatter-gather-test.xml

```diff
@@ -204,6 +204,53 @@
         </error-handler>
     </flow>
 
+    <flow name="scatterGattherInsideErrorHandler">
+        <raise-error type="APP:SOME"/>
+        <error-handler>
+            <on-error-continue>
+                <scatter-gather>
+                    <route>
+                        <logger level="ERROR" message="Hello World"/>
+                    </route>
+                    <route>
+                        <logger level="ERROR" message="Hello World"/>
+                    </route>
+                </scatter-gather>
+                <set-payload value="hello"/>
+            </on-error-continue>
+        </error-handler>
+    </flow>
+
+    <flow name="scatterGattherInsideErrorHandlerThrowsError">
+        <try>
+            <raise-error type="APP:SOME"/>
+            <error-handler>
+                <on-error-continue>
+                    <scatter-gather>
+                        <route>
+                            <logger level="ERROR" message="Hello World"/>
```

Comment:
add a verification that this was actually executed

Reply:
It's down, in the error handler (it validates and also sets the payload that it's asserted in the test case)